bazel/cargo/wasmtime/remote/BUILD.rustix-0.35.13.bazel

```diff
@@ -163,36 +163,20 @@ rust_library(
             "@rules_rust//rust/platform:x86_64-unknown-linux-gnu",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
-        ): [
-            "@wasmtime__linux_raw_sys__0_0_46//:linux_raw_sys",
-        ],
-        "//conditions:default": [],
-    }) + selects.with_or({
-        # cfg(any(rustix_use_libc, miri, not(all(target_os = "linux", any(target_arch = "x86", all(target_arch = "x86_64", target_pointer_width = "64"), all(target_endian = "little", any(target_arch = "arm", all(target_arch = "aarch64", target_pointer_width = "64"), target_arch = "powerpc64", target_arch = "riscv64", target_arch = "mips", target_arch = "mips64")))))))
-        (
-            "@rules_rust//rust/platform:i686-unknown-linux-gnu",
-            "@rules_rust//rust/platform:x86_64-unknown-linux-gnu",
-            "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
-            "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
```

Comment:
This shouldn't be removed. That's why you're seeing all those errors about missing `linux_raw_sys`.

Always try to verify the diff between the old and new version of the patch. That's the easiest way to spot issues.